src/CodeCompliance/Domain/Checks/PrivateApi/Used/AbstractUsedCodeComplianceCheck.php

```diff
@@ -142,19 +142,6 @@ protected function getMethodBody(ReflectionMethod $method): ?string
         return implode('', array_slice($sourceFile, $startLine, $length));
     }
 
-    /**
-     * @param array<string> $coreNamespaces
-     * @param array<string> $classNamespaceList
-     *
-     * @return array<string>
-     */
-    protected function filterCoreClasses(array $coreNamespaces, array $classNamespaceList): array
-    {
-        return array_filter($classNamespaceList, function ($classNamespace) use ($coreNamespaces) {
-            return $this->hasCoreNamespace($coreNamespaces, $classNamespace);
-        });
-    }
-
```

Comment:
public method getParamNamespacesFromDocComment should be placed above protected

Reply:
Thanks ++